src/lib/Messages.js

```diff
@@ -255,67 +255,67 @@ class Messages {
     }
   };
 
-  tryfromBinary = <TType>(buffer: Buffer, typeName: string): ?TType => {
-      var result = null;
+  tryFromBinary = (buffer: Buffer, typeName: string): ?mixed => {
+      let result = null;
       try {
         result = this.fromBinary(buffer, typeName);
-      } catch (exception) {
-        logger.error('Could not parse type: ${typeName} ${buffer}', exception);
+      } catch (error) {
+        logger.error('Could not parse type: ${typeName} ${buffer}', error);
       }
       return result;
   };
 
-  fromBinary = <TType>(buffer: Buffer, typeName: string): TType => {
```

Comment:
now we return from this function: 1) function calls with unknown types: ( bufferReader.shiftFloat() etc) 2) buffer 3) string ..with TType I get Flow error:

"Buffer / string : This type is incompatible with the expected return type of
some incompatible instantiation of `TType`"

in last two cases.